src/NinjaTools.FluentMockServer.API/Extensions/Responses/OkResponse.cs

```diff
@@ -0,0 +1,23 @@
+using JetBrains.Annotations;
+
+namespace NinjaTools.FluentMockServer.API.Extensions.Responses
+{
+    /// <inheritdoc />
+    public class OkResponse<T> : Response<T>
+    {
+        /// <inheritdoc />
+        public OkResponse([NotNull] T data) : base(data)
+        {
+        }
+    }
+    
+    
+    /// <inheritdoc />
+    public class OkResponse : Response
+    {
+        /// <inheritdoc />
+        public OkResponse()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/app/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4722826)